src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -182,6 +182,38 @@ object Scanners {
      */
     var revComments: List[Comment] = Nil
```

Comment:
Thanks for the LGTM @DarkDimius - regarding this `revComments` variable - should I delete it? I can not see any purpose it might serve now...